src/tactic/interactive_expr.lean

```diff
@@ -300,13 +300,13 @@ tc.mk_simple
     gs ← get_goals,
     hs ← gs.mmap (λ g, do set_goals [g], flip tc.to_html ft $ tactic_view_goal local_c target_c),
     set_goals gs,
-    let goal_message :=
+    let goal_message : html γ :=
       if gs.length = 0 then
-        "goals accomplished 🎉"
+        h "div" [cn "f5"] ["goals accomplished 🎉"]
       else if gs.length = 1 then
         "1 goal"
```

Comment:
`html.of_string` here too?

Reply:
There is a coercion for this.  I don't know why it didn't work below.

Reply:
I see, thanks!